cpp/src/kmeans/kmeans_mg_impl.cuh

```diff
@@ -116,8 +116,8 @@ void initKMeansPlusPlus(const raft::handle_t &handle,
   auto n_samples = X.getSize(0);
   auto n_features = X.getSize(1);
   auto n_clusters = params.n_clusters;
-  ML::Distance::DistanceType metric =
-    static_cast<ML::Distance::DistanceType>(params.metric);
+  raft::distance::DistanceType metric =
```

Comment:
Can you create an issue for KMeans to use this enum as well; proliferating from the python layer, it seems KMeans is assigning an integer there but let's start having a unified structure here

Reply:
I believe this is covered by #563. I just added a note there to more explicitly call out what is needed.